pkg/dockerutil/dockerutils.go

```diff
@@ -44,6 +44,8 @@ import (
 
 // NetName provides the default network name for ddev.
 const NetName = "ddev_default"
+const requiredDockerVersion = ">= 24.0.0"
+const requiredDockerAPIVersion = ">= 1.43"
```

Comment:
This could be named `minimalDockerAPIVersion = "1.43"`, and built-in check from Docker library can be used for this (see `templateCanUse` func).